python/ribasim_testmodels/ribasim_testmodels/invalid.py

```diff
@@ -162,3 +165,67 @@ def invalid_unstable_model() -> Model:
         model.edge.add(model.basin[1 + id_shift * i], model.pump[2 + id_shift * i])
         model.edge.add(model.pump[2 + id_shift * i], model.terminal[3 + id_shift * i])
     return model
+
+
+def invalid_priorities_model() -> Model:
+    """Model with allocation active but missing priority parameter(s)."""
```

Comment:
Could you make this model as small as possible? That will make it easier to maintain the test models.